src/crypto/crypto_aes.cc

```diff
@@ -130,7 +130,14 @@ WebCryptoCipherStatus AES_Cipher(
   ByteSource buf = ByteSource::Allocated(data, buf_len);
   unsigned char* ptr = reinterpret_cast<unsigned char*>(data);
 
-  if (!EVP_CipherUpdate(
+  // Only `ubi81_sharedlibs_openssl111fips_x64` failed when `in.size()` is zero.
+  // So we regard 0 as special and DO NOT go into `EVP_CipherUpdate()` logic
+  // because it will occur failure under `ubi81_sharedlibs_openssl111fips_x64`.
+  //
+  // Refs: https://github.com/nodejs/node/pull/38913#issuecomment-866505244
```

Comment:
I don't know if it's possible to work out which version of OpenSSL 1.1.1 changed the behavior without going through and build/testing against OpenSSL 1.1.1c, 1.1.1d, 1.1.1e etc. but the comment would be a lot better if we could refer to an OpenSSL release rather than a container specific to our build CI.

Reply:
I've updated the comment and you may review again.

Reply:
@richardlau btw, I think we should leave at least one outdated OpenSSL CI to discover similar problems.